EIPS/eip-7642.md

```diff
@@ -26,20 +26,11 @@ The serving node will regenerate roughly 530GB of bloom filters (2.3B txs * 256
 These 530GBs are send over the network to the syncing peer, the syncing peer will verify them and not store them either.
 This adds an additional 530GB of unnecessary bandwidth to every sync.
 
-Additionally we propose to remove fields and messages that were deprecated by the merge, such as 
-
-- Removing the `TD` field in the `Status` message. 
-
-- Removing the `NewBlockHashes` message.
-
-- Removing the `NewBlock` message.
+Additionally we propose to remove the field that was deprecated by the merge, namely
+the `TD` field in the `Status` message. 
```

Comment:
Could make consistent with capitalisation of `td` / `TD`

Reply:
Here I simply kept the pre-existing capitalization.